app/ide-desktop/package.json

```diff
@@ -39,6 +39,7 @@
     "watch": "npm run watch --workspace enso-content",
     "watch-dashboard": "npm run watch --workspace enso-dashboard",
     "build-dashboard": "npm run build --workspace enso-dashboard",
+    "test": "npm run test --workspace enso-dashboard",
```

Comment:
Would be nice to have this integrated into the CI potentially.

Reply:
will probably do it in a separate PR because:
- it's a different part of the code so needs different reviewers
- it will be easy to revert in case it causes problems again ( :( )

Reply:
👍